
# [CVE-2021-20275](https://bugzilla.redhat.com/show_bug.cgi?id=1936666)

## Description

- `A flaw was found in privoxy before 3.0.32. A invalid read of size two may occur in chunked_body_is_complete() leading to denial of service.`

## Cvss Data

- **Access Vector**:
  - NETWORK
- **Base Score**:
  - 5.0

## Scores

- **Exploitability Score**:
  - 10.0
- **Impact Score**:
  - 2.9
- **Base Severity**:
  - MEDIUM

## Other Information

- **Publish Date**:
  - 2021-03-09 14:15:12
- **Vulnerability Status**:
  - Analyzed

## References

- **secalert@redhat.com**: https://bugzilla.redhat.com/show_bug.cgi?id=1936666
- **secalert@redhat.com**: https://lists.debian.org/debian-lts-announce/2021/03/msg00009.html
- **secalert@redhat.com**: https://security.gentoo.org/glsa/202107-16
- **secalert@redhat.com**: https://www.privoxy.org/announce.txt
